deploy/sre-prometheus/100-machine-api.PrometheusRule.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
     prometheus: sre-machine-api-node-mismatch
     role: alert-rules
   name: sre-machine-api-node-mismatch
-  namespace: openshift-monitoring
+  namespace: openshift-machine-api
```

Comment:
/hold
All rules land in openshift-monitoring.  Not sure this will work if we deploy to a different namespace.
The `namespace` label will be set by the exporter, see https://github.com/openshift/managed-prometheus-exporter-machine-api/blob/master/monitor/main.py